public/app/features/dashboard-scene/settings/VariablesEditView.tsx

```diff
@@ -1,30 +1,145 @@
 import React from 'react';
 
 import { PageLayoutType } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase } from '@grafana/scenes';
+import { SceneComponentProps, SceneObjectBase, SceneObjectRef, SceneVariables, sceneGraph } from '@grafana/scenes';
 import { Page } from 'app/core/components/Page/Page';
 
+import { DashboardScene } from '../scene/DashboardScene';
 import { NavToolbarActions } from '../scene/NavToolbarActions';
-import { getDashboardSceneFor } from '../utils/utils';
 
 import { DashboardEditView, DashboardEditViewState, useDashboardEditPageNav } from './utils';
-
-export interface VariablesEditViewState extends DashboardEditViewState {}
+import { VariableEditorList } from './variables/VariableEditorList';
+export interface VariablesEditViewState extends DashboardEditViewState {
+  dashboardRef: SceneObjectRef<DashboardScene>;
+}
 
 export class VariablesEditView extends SceneObjectBase<VariablesEditViewState> implements DashboardEditView {
+  public static Component = VariableEditorSettingsListView;
+
   public getUrlKey(): string {
     return 'variables';
   }
 
-  static Component = ({ model }: SceneComponentProps<VariablesEditView>) => {
-    const dashboard = getDashboardSceneFor(model);
-    const { navModel, pageNav } = useDashboardEditPageNav(dashboard, model.getUrlKey());
+  private get _dashboard(): DashboardScene {
+    return this.state.dashboardRef.resolve();
+  }
+
+  public getDashboard(): DashboardScene {
+    return this._dashboard;
+  }
+
+  private get _variablesScene(): SceneVariables {
+    return sceneGraph.getVariables(this.getDashboard());
+  }
+
+  public getVariablesScene(): SceneVariables {
+    return this._variablesScene;
+  }
```

Comment:
```suggestion
  public getVariableSet(): SceneVariables {
    return sceneGraph.getVariables(this.getDashboard());
  }
```

think this is more clear than the word `getVariablesScene` . nitpick: Having two identical functions (one getter and one plain function) seems a bit unnessary.